.drone.yml

```diff
@@ -231,6 +231,24 @@ steps:
     - drone-publish.rancher.io
     status:
     - failure
+- name: docker-publish-rancherd
+  image: plugins/docker
+  settings:
+    build_args:
+      - ARCH=amd64
+      - "TAG=${DRONE_TAG}"
+    context: cmd/rancherd/package
+    custom_dns: 1.1.1.1
+    dockerfile: cmd/rancherd/package/Dockerfile
```

Comment:
if you have this in a drone pipeline do you need the extra docker build in scripts/package?

Reply:
removed